data/korath/korath variants.txt

```diff
@@ -857,7 +857,30 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
 		"Steering (Planetary Class)"
 		"Jump Drive"
 
+ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Crippled)"
+	outfits
+		"Langrage Hyper-Heaver" 3
+		"Warder Anti-Missile" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor" 5
+		"Microbot Defense Station" 6
+		"Thermal Repeater Rifle" 551
```

Comment:
You gave rifles to the children?
I warn you, I'll shoot...

Reply:
Oh and whilst we're at it add actual children to the ship.
Before outfits, on an added line, add "crew" 500

Reply:
Or maybe its best to do it with an invisible outfit that adds required crew actually (invisible as in without category)? I don't have strong opinions on this and since we dont cap it that should be a fine solution.

Reply:
The children aren't crew though, and might actually take up less than one bunk. In my head it would make sense to leave them out of all the crew counts. Though yeah, less guns more defense station makes sense either way.

Reply:
Sounds good to me, 331 is the default amount of thermal rifles for this ship btw